app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.test.tsx

```diff
@@ -147,8 +152,17 @@ describe("AppLayoutWithSidebar", () => {
       expect(screen.queryByTestId("nav-request-review")).toBeInTheDocument();
     });
 
-    it("when in review, disables all edit page links", async () => {
-      render(<Subject status={NimbusExperimentStatus.REVIEW} />);
+    it("when in preview, disables all edit page links", async () => {
+      render(<Subject status={NimbusExperimentStatus.PREVIEW} />);
+
+      // In preview these should all not be <a> tags, but instead <span>s
+      ["overview", "branches", "metrics", "audience"].forEach((slug) => {
+        expect(screen.getByTestId(`nav-edit-${slug}`).tagName).toEqual("SPAN");
+      });
+    });
+
+    it("when in (publish status) review, disables all edit page links", async () => {
+      render(<Subject publishStatus={NimbusExperimentPublishStatus.REVIEW} />);
```

Comment:
I think we'll want this also to be true for APPROVED and WAITING publish status - i.e. everything but IDLE.

Reply:
Yeah, that is consistent with how we've arranged our header pills. I'll add more tests 👍